From 745c42555077606c05f0c95d6fb29bf713a85e43 Mon Sep 17 00:00:00 2001
From: local-build <local@build>
Date: Wed, 8 Apr 2026 12:31:42 -0500
Subject: [PATCH 2/2] Makefile

---
 Makefile | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index ea2d55caa..dfbb489f5 100644
--- a/Makefile
+++ b/Makefile
@@ -15,10 +15,12 @@ BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 REGISTRY_AND_USERNAME := $(IMAGE_REGISTRY)/$(USERNAME)
 NAME = Talos
 
+SED ?= sed
+
 CLOUD_IMAGES_EXTRA_ARGS ?= ""
 ZSTD_COMPRESSION_LEVEL ?= 18
 
-CI_RELEASE_TAG := $(shell git log --oneline --format=%B -n 1 HEAD^2 -- 2>/dev/null | head -n 1 | sed -r "/^release\(.*\)/ s/^release\((.*)\):.*$$/\\1/; t; Q")
+CI_RELEASE_TAG := $(shell git log --oneline --format=%B -n 1 HEAD^2 -- 2>/dev/null | head -n 1 | $(SED) -r "/^release\(.*\)/ s/^release\((.*)\):.*$$/\\1/; t; Q")
 
 ARTIFACTS := _out
 
@@ -97,7 +99,7 @@ GO_VERSION ?= 1.26
 # renovate: datasource=npm depName=markdownlint-cli
 MARKDOWNLINTCLI_VERSION ?= 0.48.0
 OPERATING_SYSTEM := $(shell uname -s | tr "[:upper:]" "[:lower:]")
-ARCH := $(shell uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')
+ARCH := $(shell uname -m | $(SED) 's/x86_64/amd64/' | $(SED) 's/aarch64/arm64/')
 TALOSCTL_DEFAULT_TARGET := talosctl-$(OPERATING_SYSTEM)
 TALOSCTL_EXECUTABLE := $(PWD)/$(ARTIFACTS)/$(TALOSCTL_DEFAULT_TARGET)-$(ARCH)
 INTEGRATION_TEST := integration-test
@@ -724,7 +726,7 @@ reproducibility-test-docker-%: $(ARTIFACTS)
 	@mkdir -p _out1/ _out2/
 	@$(MAKE) docker-$* DEST=_out1/
 	@$(MAKE) docker-$* DEST=_out2/ TARGET_ARGS="--no-cache"
-	@find _out1/ -type f | xargs -IFILE diffoscope FILE `echo FILE | sed 's/_out1/_out2/'`
+	@find _out1/ -type f | xargs -IFILE diffoscope FILE `echo FILE | $(SED) 's/_out1/_out2/'`
 	@rm -rf _out1/ _out2/
 
 reproducibility-test-local-%: $(ARTIFACTS)
@@ -732,7 +734,7 @@ reproducibility-test-local-%: $(ARTIFACTS)
 	@mkdir -p _out1/ _out2/
 	@$(MAKE) local-$* DEST=_out1/
 	@$(MAKE) local-$* DEST=_out2/ TARGET_ARGS="--no-cache"
-	@find _out1/ -type f | xargs -IFILE diffoscope FILE `echo FILE | sed 's/_out1/_out2/'`
+	@find _out1/ -type f | xargs -IFILE diffoscope FILE `echo FILE | $(SED) 's/_out1/_out2/'`
 	@rm -rf _out1/ _out2/
 
 reproducibility-test-iso: $(ARTIFACTS)
-- 
2.50.1 (Apple Git-155)

